extensions/rest-raml/src/types.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/rest-raml
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+/**
+ * Interface defining the component's options object
+ */
+export interface RestRamlComponentOptions {
+  // Add the definitions here
+  httpPath?: string;
```

Comment:
Please add a tsdoc comment to explain what is the `httpPath` option for? When would the users want to change from the default and what value they should provide?